src/ContentCreator.php

```diff
@@ -782,6 +785,10 @@ public function importCsvFiles($keep_content_files = FALSE) {
             }
           }
 
+          if (PHP_SAPI === 'cli') {
```

Comment:
This could be changed to a class exists to test if Drush is being used.

Reply:
Yes, will do that